### PR TITLE
ImageViewer: Visual improvements

### PR DESCRIPTION
**Pull request by SebastianAigner**
Adds buttons and margins according to the original Figma designs, minor layout changes.

Now crossfades between Gallery and Memory View (illusion of keeping the header on screen)

https://user-images.githubusercontent.com/2178959/223845723-9461b324-d919-42d8-bf34-8dcc2f6d5afd.MP4

### CODE REVIEW

**Comment by SebastianAigner** [comment]
Seems the fade animation has some issues with UIKit interop 🤔 This is what it looks like on desktop (where it's just a static image instead of a map component)
https://user-images.githubusercontent.com/2178959/223845951-806c34ac-e1df-4dd5-a582-707e3f39df1a.mov

File: experimental/examples/imageviewer/shared/src/desktopMain/kotlin/example/imageviewer/view/ImageViewer.desktop.kt [hunk 1]
```diff
@@ -58,7 +58,7 @@ fun ApplicationScope.ImageViewerDesktop() {
         title = "Image Viewer",
         state = WindowState(
             position = WindowPosition.Aligned(Alignment.Center),
-            size = getPreferredWindowSize(800, 1000)
+            size = getPreferredWindowSize(720, 857)
```
Comment (dima-avdeev-jb) [inline experimental/examples/imageviewer/shared/src/desktopMain/kotlin/example/imageviewer/view/ImageViewer.desktop.kt:61]: Why this size?

**Review by dima-avdeev-jb — APPROVED** [review]

File: experimental/examples/imageviewer/shared/src/desktopMain/kotlin/example/imageviewer/view/ImageViewer.desktop.kt (hunk 1 above)
Comment (SebastianAigner) [reply experimental/examples/imageviewer/shared/src/desktopMain/kotlin/example/imageviewer/view/ImageViewer.desktop.kt:61]: Height is equivalent to iOS, I tried to find a width that visually seems harmonic on desktop.

File: experimental/examples/imageviewer/shared/src/commonMain/kotlin/example/imageviewer/view/MemoryScreen.kt [hunk 2]
```diff
@@ -212,7 +238,7 @@ internal fun RelatedMemoriesVisualizer(
 ) {
     Box(
         modifier = Modifier.padding(10.dp, 0.dp).clip(RoundedCornerShape(10.dp)).fillMaxWidth()
-            .height(200.dp)
+        //.height(200.dp)
```
Comment (dima-avdeev-jb) [inline experimental/examples/imageviewer/shared/src/commonMain/kotlin/example/imageviewer/view/MemoryScreen.kt:241]:
```suggestion

```
Can we remove it?

**Review by dima-avdeev-jb — APPROVED** [review]

**Pull request merged**